woocommerce/woocommerce-android
# Fix clear menu item visibility in the product filters

Closes: #7465


### Description

The problem described in the issue happens because "onCreateMenu" is called after the visibility is updated and new menu is created. The PR makes the update of  the visibility in "onPrepareMenu"

### Testing instructions

* Products
* Filters 
* Select filter
* Go back
* Notice that clear button there and it works

### Images/gif


https://user-images.githubusercontent.com/4923871/206129850-f0dde5a0-2927-4831-a2b5-3b76b05f99bb.mp4
